web/docs/guides/with-sveltekit.mdx

```diff
@@ -0,0 +1,516 @@
+
+export const user = writable(false);
+```
+
+
+### Account page
+
+After a user is signed in we can allow them to edit their profile details and manage their account.
+Let's create a new component for that called `Profile.svelte`.
+
+```html title="lib/Profile.svelte"
+<script>
+  import { supabase } from '$lib/supabaseClient'
+  import { user } from '$lib/sessionStore'
+
+  let loading = true
+  let username = null
+  let website = null
+  let avatar_url = null
+
+  async function getProfile() {
+    try {
+      loading = true
+      const user = supabase.auth.user()
+
+      let { data, error, status } = await supabase
+        .from('profiles')
+        .select(`username, website, avatar_url`)
+        .eq('id', user.id)
+        .single()
+
+      if (error && status !== 406) throw error
+
+      if (data) {
+        username = data.username
+        website = data.website
+        avatar_url = data.avatar_url
+      }
+    } catch (error) {
+      alert(error.message)
+    } finally {
+      loading = false
+    }
+  }
+
+  async function updateProfile() {
+    try {
+      loading = true
+      const user = supabase.auth.user()
+
+      const updates = {
+        id: user.id,
+        username,
+        website,
+        avatar_url,
+        updated_at: new Date(),
+      }
+
+      let { error } = await supabase.from('profiles').upsert(updates, {
+        returning: 'minimal', // Don't return the value after inserting
+      })
+
+      if (error) throw error
+    } catch (error) {
+      alert(error.message)
+    } finally {
+      loading = false
+    }
+  }
+
+  async function signOut() {
+    try {
+      loading = true
+      let { error } = await supabase.auth.signOut()
+      if (error) throw error
+    } catch (error) {
+      alert(error.message)
+    } finally {
+      loading = false
+    }
+    
+  }
+</script>
+
+<form use:getProfile class="form-widget" on:submit|preventDefault={updateProfile}>
+  <div>
+    <label for="email">Email</label>
+    <input id="email" type="text" value={$user.email} disabled />
+  </div>
+  <div>
+    <label for="username">Name</label>
+    <input
+      id="username"
+      type="text"
+      bind:value={username}
+    />
+  </div>
+  <div>
+    <label for="website">Website</label>
+    <input
+      id="website"
+      type="website"
+      bind:value={website}
+    />
+  </div>
+
+  <div>
+    <input type="submit" class="button block primary" value={loading ? 'Loading ...' : 'Update'} disabled={loading}/>
+  </div>
+
+  <div>
+    <button class="button block" on:click={signOut} disabled={loading}>
+      Sign Out
+    </button>
+  </div>
+</form>
+```
+
+
+
+### Launch!
+
+Now that we have all the components in place, let's update `src/routes/index.svelte`:
+
+```html title="src/routes/index.svelte"
+<script>
+	import {user} from "$lib/sessionStore"
+	import {supabase} from "$lib/supabaseClient"
+	import Auth from "$lib/Auth.svelte"
+	import Profile from "$lib/Profile.svelte"
```

Comment:
These 2 imports seem to be broken. I think you need to remove the `.svelte` at the end

Reply:
Ok, I just removed them

Reply:
Hey @Isaiah-Hamilton, I just realized that removing '.svelte' doesn't fix it, I should have tested before the commit. I have no idea why this is hit and miss, but it actually won't work for me without the '.svelte'. I did hit ctrl-c and restart the dev server a few times, but it did give me the same SSR module load error at first, then it went away.  I'm not in a hurry to ship this, just trying to get to the bottom of it. This is kinda strange because, almost buggy. I have a working copy of the app here: https://github.com/timsonner/sveltekit-supabase. I'll have to go back through the guide and see what I missed, this may be a bug. Can you try ctrl-c and restarting the dev server a few times? It's not a great solution, but I can't figure out why this doesn't work, then does.

Reply:
Sorry, also in Auth.svelte I changed the import to: "import {supabase} from '$lib/supabaseClient'" instead of "import {supabase} from './supabaseClient'" this may have been the problem and it was supposed to make it into a previous commit. Kinda just got all scrambled and out of sync with my commits and multiple project windows open.